TCMSApp/src/client/app/tests/tests.controller.js

```diff
@@ -10,7 +10,13 @@
     function TestsController(logger, FakeTestsFactory, $uibModal) {
         var vm = this;
 
+        vm.stepList = [1,2];
+        vm.addStep = addStep;
+        vm.delStep = delStep;
+
         vm.trigger = 0; // suite -> testCases
```

Comment:
> Just check that you're pointing the correct index of the array

actually, 'trigger' for 'suite._id' (so current id is pointing to the current suite)